search-service/src/main/kotlin/com/egm/stellio/search/service/AttributeInstanceService.kt

```diff
@@ -1,6 +1,7 @@
 package com.egm.stellio.search.service
 
 import arrow.core.Either
+import arrow.core.computations.ResultEffect.bind
```

Comment:
this import is deprecated

Reply:
> Also, find a more explicit name for your PR (explain what it fixes)

Maybe `fix(search): wrong dates in buckets during aggregations`